agent/acs/handler/acs_handler_test.go

```diff
@@ -519,7 +519,7 @@ func TestHandlerReconnectDelayForInactiveInstanceError(t *testing.T) {
 		mockWsClient.EXPECT().Connect().Do(func() {
 			reconnectDelay := time.Now().Sub(firstConnectionAttemptTime)
 			t.Logf("Delay between successive connections: %v", reconnectDelay)
-			if reconnectDelay < inactiveInstanceReconnectDelay {
+			if reconnectDelay+1 < inactiveInstanceReconnectDelay {
```

Comment:
Rather than blindly adding 1, can you:
* Include this information about intermittent time subtraction issues in an explanatory comment
* Explicitly allow for some slop in the assertion here (possibly up to `1ms`?) by adjusting the expected time rather than the observed time and having a named value instead of a literal (use a constant or variable to describe the modifier).